Kill -Wno-error=cpp, old(ish) gcc versions (< 4.4) don't like it and we still
got 4.3.x available for internal toolchains.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura openswan-2.6.39.orig/lib/libopenswan/Makefile openswan-2.6.39/lib/libopenswan/Makefile
--- openswan-2.6.39.orig/lib/libopenswan/Makefile	2013-06-11 07:22:50.228520267 -0300
+++ openswan-2.6.39/lib/libopenswan/Makefile	2013-06-11 07:27:40.087849447 -0300
@@ -86,7 +86,7 @@
 CFLAGS+= -Wno-error=cast-qual

 # some junk left in alg_info.c
-CFLAGS+= -Wno-error=cpp
+#CFLAGS+= -Wno-error=cpp

 #CFLAGS+= -Wmissing-declarations
 CFLAGS+= -Wstrict-prototypes
diff -Nura openswan-2.6.39.orig/programs/pluto/Makefile openswan-2.6.39/programs/pluto/Makefile
--- openswan-2.6.39.orig/programs/pluto/Makefile	2013-06-11 07:22:49.753504978 -0300
+++ openswan-2.6.39/programs/pluto/Makefile	2013-06-11 07:27:53.020265681 -0300
@@ -47,7 +47,7 @@

 # must turn this off due to myid.c
 CFLAGS+= -Wno-error=cast-qual
-CFLAGS+= -Wno-error=cpp
+#CFLAGS+= -Wno-error=cpp

 ifeq ($(HAVE_BROKEN_POPEN),true)
 CFLAGS+=-DHAVE_BROKEN_POPEN
